File: 2533-bitwise-xor-of-all-pairings/bitwise-xor-of-all-pairings.py
class Solution:
    def xorAllNums(self, a: List[int], b: List[int]) -> int:
        xr = 0 
        if len(b) % 2 == 1:
            for i in a:
                xr ^= i
        
        if len(a) % 2 == 1:
            for i in b:
                xr ^= i
        
        return xr 

File: 2533-bitwise-xor-of-all-pairings/bitwise-xor-of-all-pairings.java
class Solution {
    public int xorAllNums(int[] a, int[] b) {
        int xr = 0 ;
        if(b.length % 2 == 1){
            for(int i : a)
                xr ^= i ;
        }
        if(a.length % 2 == 1){
            for(int i : b)
                xr ^= i ;
        }
        return xr ;
    }
}